edx_event_bus_kafka/internal/consumer.py

```diff
@@ -156,8 +150,8 @@ def emit_signals_from_message(self, msg):
         # CloudEvents specifies using UTF-8 for header values, so let's be explicit.
         event_type_str = event_type.decode("utf-8")
 
-        # If we get a message with the wrong signal encoding, we do not want to send it along.
-        # TODO (EventBus): Handle this particular sad path more gracefully.
+        # TODO: Accept multiple event types: https://github.com/openedx/openedx-events/issues/78
+        # TODO: Maybe raise error here? Or at least set a metric or custom attribute.
```

Comment:
Not sure how to handle this in terms of comments or notes in some task, but:
- Should all of these error logs + returns just raise an error instead? Then we could log and send the error to New Relic.

Reply:
Probably. I can fold that into the error-handling ticket(s).

Reply:
I've listed that in https://github.com/edx/edx-arch-experiments/issues/55 and will break it out later.